chart/templates/dag-processor/dag-processor-deployment.yaml

```diff
@@ -130,6 +130,9 @@ spec:
         {{- if .Values.dagProcessor.extraInitContainers }}
         {{- toYaml .Values.dagProcessor.extraInitContainers | nindent 8 }}
         {{- end }}
+{{- if .Values.dagProcessor.waitForMigrations.env }}
+{{ tpl (toYaml .Values.dagProcessor.waitForMigrations.env) $ | indent 12 }}
+{{- end }}
       containers:
         - name: dag-processor
```

Comment:
You've missed adding `dagProcessor.env` to the main container here.

Reply:
Ah thanks! Added the snippet